images/runtime/dotnetcore/3.0/Dockerfile

```diff
@@ -0,0 +1,35 @@
+# Startup script generator
+FROM golang:1.11-stretch as startupCmdGen
+# GOPATH is set to "/go" in the base image
+WORKDIR /go/src
+COPY src/startupscriptgenerator/src .
+ARG GIT_COMMIT=unspecified
+ARG BUILD_NUMBER=unspecified
+ENV GIT_COMMIT=${GIT_COMMIT}
+ENV BUILD_NUMBER=${BUILD_NUMBER}
+RUN ./build.sh dotnetcore /opt/startupcmdgen/startupcmdgen
+
+FROM mcr.microsoft.com/dotnet/core/sdk:3.0 AS tools-install
+RUN dotnet tool install --tool-path /dotnetcore-tools dotnet-sos
+RUN dotnet tool install --tool-path /dotnetcore-tools dotnet-trace
+RUN dotnet tool install --tool-path /dotnetcore-tools dotnet-dump
+RUN dotnet tool install --tool-path /dotnetcore-tools dotnet-counters
+
+FROM mcr.microsoft.com/dotnet/core/aspnet:3.0.0
```

Comment:
In this PR we are only installing the necessary tools/packages but are NOT enabling capturing dumps. The idea is for AppService to provide the values for the following .NET Core settings when they derive from our image.
For example, AppService can give the default directory for dumps to be the `$HOME/Logs/CoreDumps`

```
ENV COMPlus_DbgEnableMiniDump="1" (This enables automatic capturing of coredump before a program crashes finally)
ENV COMPlus_DbgMiniDumpName="$HOME/Logs/Dumps/coredump.%d" (This is for the location of the dumps)
```